Ambermoon.net/GameWindow.cs

```diff
@@ -1187,7 +1187,8 @@ void Window_Load()
             initialized = true;
 
             var additionalData = OperatingSystem.IsMacOS() ? new Dictionary<string, BinaryReader>() : AdditionalData.Loader.Load();
-            var builtinVersionReader = OperatingSystem.IsMacOS() ? new BinaryReader(File.OpenRead("versions.dat")) :
+            const string versions_filename = "versions.dat";
```

Comment:
Can you please use `VersionsFilename`. I like the PascalCase convention for constants in C#. It would be consistent with the rest of the project (hopefully).